collector/process.go

```diff
@@ -190,6 +190,14 @@ func (c *processCollector) Collect(ctx *ScrapeContext, ch chan<- prometheus.Metr
 		log.Debugf("Could not query WebAdministration namespace for IIS worker processes: %v. Skipping", err)
 	}
 
+	// Query service list to provide display name for processess if it is part of a service.
+	var dst_svc []Win32_Service
```

Comment:
I'm a bit concerned about the cost of this query (the service collector is already slow, and adding the same query here doesn't help). It'd be neat if we could get this from perflib instead (and for the service collector as well). Would you be able to dig into finding a way to do that?